app/assets/stylesheets/components/_contextual-guidance.scss

```diff
@@ -1,3 +1,6 @@
+@import "../../../node_modules/govuk-frontend/settings/all";
+@import "../../../node_modules/govuk-frontend/helpers/all";
+
```

Comment:
I wonder if we can do anything govuk_publishing_components to make these be in a more logical place - they caught me out for a while the other day when I realised it was from the gems node_modules directory rather than ours, and I think they only work off a bit of a hack working up from the gems app/assets directory.

Reply:
Yes, they come from the gem and is not clear. I'll think about it